kolibri/core/assets/src/views/core-base.vue

```diff
@@ -110,6 +115,7 @@
       },
     },
     mounted() {
+      this.updateDocumentTitle(this.title);
```

Comment:
The other thing we need to do is get these titles into translatable strings - they are mostly not wrapped as of now.